.github/workflows/main.yml

```diff
@@ -108,6 +108,29 @@ jobs:
       - name: Test Release
         run: ./build/release/test/all_tests
 
+  test_windows:
+    name: "Test (windows-latest, msvc)"
+    runs-on: windows-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v4
+        with:
+          submodules: recursive
+          fetch-tags: true
+          fetch-depth: 50
```

Comment:
```suggestion
      - name: Checkout Code
        uses: actions/checkout@v4
        with:
          submodules: recursive
          fetch-tags: true
```

Reply:
Ah yea, this can be removed now. Ty for catching that.